x-pack/platform/plugins/shared/fleet/common/settings/agent_policy_settings.tsx

```diff
@@ -195,5 +203,49 @@ export const AGENT_POLICY_ADVANCED_SETTINGS: SettingsConfig[] = [
     learnMoreLink:
       'https://www.elastic.co/guide/en/fleet/current/elastic-agent-standalone-logging-config.html#elastic-agent-standalone-logging-settings',
     schema: zodStringWithDurationValidation,
+    example_value: '10m',
+  },
+  {
+    name: 'agent.monitoring._runtime_experimental',
+    title: i18n.translate('xpack.fleet.settings.agentPolicyAdvanced.monitoringRuntimeTitle', {
+      defaultMessage: 'Monitoring Runtime (experimental)',
+    }),
+    description: (
+      <FormattedMessage
+        id="xpack.fleet.settings.agentPolicyAdvanced.monitoringRuntimeDescription"
+        defaultMessage="Change how the Beat inputs used for Elastic Agent self-monitored are executed."
+      />
+    ),
+    api_field: {
+      name: 'agent_monitoring_runtime_experimental',
+    },
+    schema: z.enum(['', 'process', 'otel']).default(''),
+    example_value: 'otel',
+    options: [
+      {
+        value: '',
+        text: i18n.translate(
+          'xpack.fleet.settings.agentPolicyAdvanced.monitoringRuntimeDefaultLabel',
+          {
+            defaultMessage: 'Default',
+          }
+        ),
+      },
+      {
+        value: 'process',
+        text: i18n.translate(
+          'xpack.fleet.settings.agentPolicyAdvanced.monitoringRuntimeProcessLabel',
+          {
+            defaultMessage: 'Process',
+          }
+        ),
+      },
+      {
+        value: 'otel',
+        text: i18n.translate('xpack.fleet.settings.agentPolicyAdvanced.monitoringRuntimeLabel', {
```

Comment:
```suggestion
        text: i18n.translate('xpack.fleet.settings.agentPolicyAdvanced.monitoringRuntimeOtelLabel', {
```

extremely small nit with naming of this i18n key 😅